snippets/odoo-python-snippets.cson

```diff
@@ -44,15 +44,41 @@
 
 	'Many2one Field':
 		'prefix': 'oo_field_many2one'
-		'body': 'new_field_id = fields.Many2one(comodel_name="$1", string="$2", )'
+		"body": '''
+			Many2one(
+				"${10:res.partner}"${14:,
```

Comment:
Actually the code you provide means that they are supposed to be positional, but they are not required.